<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PixelSpreadsheet</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>myapp.png</normaloff>myapp.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="imageBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>306</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>Image:</string>
    </property>
    <widget class="QLabel" name="imageView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>256</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>test.jpg</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="loadImage">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>271</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Image Information</string>
      </property>
     </widget>
     <widget class="QLabel" name="outputRows">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Rows: </string>
      </property>
     </widget>
     <widget class="QLabel" name="originalSize">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Original Size: </string>
      </property>
     </widget>
     <widget class="QLabel" name="shownSize">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Shown Size: </string>
      </property>
     </widget>
     <widget class="QLabel" name="excelCompat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Excel 2002: </string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>681</width>
        <height>61</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="saveImage">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>410</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Image</string>
     </property>
    </widget>
    <widget class="QLabel" name="outputRows_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Loaded in memory:</string>
     </property>
    </widget>
    <widget class="QLabel" name="outputRows_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Saved to disk:</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_memory_pos">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>370</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/new/prefix1/checked.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="image_memory_neg">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>370</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/new/prefix1/xed.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="image_save_pos">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>390</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/new/prefix1/checked.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="image_save_neg">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>390</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/new/prefix1/xed.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="spreadsheetBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>306</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>Spreadsheet</string>
    </property>
    <widget class="QPushButton" name="loadSpreadsheet">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Sheet</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>271</width>
       <height>111</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Spreadsheet Info</string>
      </property>
     </widget>
     <widget class="QLabel" name="resolution">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Image: </string>
      </property>
     </widget>
     <widget class="QLabel" name="red">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Red:</string>
      </property>
     </widget>
     <widget class="QLabel" name="green">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Green:</string>
      </property>
     </widget>
     <widget class="QLabel" name="blue">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Blue:</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="saveSheet">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>410</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Sheet</string>
     </property>
    </widget>
    <widget class="QLabel" name="outputRows_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Loaded in memory:</string>
     </property>
    </widget>
    <widget class="QLabel" name="outputRows_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Saved to disk:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ss_memory_pos">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>360</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/new/prefix1/checked.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="ss_memory_neg">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>360</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/new/prefix1/xed.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="ss_save_pos">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>380</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/new/prefix1/checked.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="ss_save_neg">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>380</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/new/prefix1/xed.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>5</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>47</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <item row="0" column="0">
      <property name="text">
       <string>x</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>y</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>r</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>g</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>b</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>255</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>255</string>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string>255</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>255</string>
      </property>
     </item>
     <item row="2" column="3">
      <property name="text">
       <string>255</string>
      </property>
     </item>
     <item row="2" column="4">
      <property name="text">
       <string>255</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item row="3" column="2">
      <property name="text">
       <string>255</string>
      </property>
     </item>
     <item row="3" column="3">
      <property name="text">
       <string>255</string>
      </property>
     </item>
     <item row="3" column="4">
      <property name="text">
       <string>255</string>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item row="4" column="2">
      <property name="text">
       <string>255</string>
      </property>
     </item>
     <item row="4" column="3">
      <property name="text">
       <string>255</string>
      </property>
     </item>
     <item row="4" column="4">
      <property name="text">
       <string>255</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="imageToSpreadsheet">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Image to Spreadsheet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="spreadsheetToImage">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Spreadsheet to Image</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="shrink">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Shrink</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_to_ss_image">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/new/prefix1/reverse.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ss_to_image_image">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/new/prefix1/ConverterSymbols.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>531</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/new/prefix1/Banner.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="loading_status_image">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>-30</y>
      <width>831</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/new/prefix1/loading.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
